packages/components/pie-card/src/defs.ts

```diff
@@ -58,3 +60,12 @@ export interface CardProps {
      */
     padding?: PaddingValue | `${PaddingValue},${PaddingValue}`;
 }
+
+export type DefaultProps = ComponentDefaultPropsGeneric<CardProps, 'tag' | 'variant' | 'disabled' | 'isDraggable'>;
+
+export const defaultProps: Readonly<DefaultProps> = {
```

Comment:
I notice that we're using `Readonly` here. It is also applied in the definition of `ComponentDefaultPropsGeneric`: https://github.com/justeattakeaway/pie/pull/1451/files#diff-1e0d8ad270e50cba2b32b4be90b4ae7916c97dd38197a058ded70b2e05fda04d

Is it necessary to have `Readonly` in both places?

Reply:
Fixed the leftovers 👍